test/e2e/README.md

```diff
@@ -115,99 +101,22 @@ If you need to add a non existing public container in the workflow, create it in
 
 But, keep in mind this become an additional piece of software to maintain.
 
-
-
 # Upgrade - bump
 
 This section helps you to upgrade any part of the end to end testing.
 
 The current end to end testing pipeline relies on:
-* [Pupernetes](https://github.com/DataDog/pupernetes)
 * [Argo](https://github.com/argoproj/argo)
-* [Kinvolk Flatcar](https://www.flatcar-linux.org)
-
-## Bump hyperkube version
 
-Read the command lines docs of [pupernetes](https://github.com/DataDog/pupernetes/tree/master/docs)
+Upgrade Argo version by changing version in `test/e2e/scripts/run-instance/20-argo-download.sh` and setting new checksum value in `test/e2e/scripts/run-instance/argo.sha512sum`
 
-In the Ignition *systemd.units* list, find the `pupernetes.service` one.
+* [Kind](https://kind.sigs.k8s.io/)
 
-In its content, the systemd unit has a `[service]` directive where there is a single `ExecStart=` field: you need to add / edit the `--hyperkube-version` flag in the command line.
-
-The value of this flag must be a valid release such as `1.10.1`: `--hyperkube-version=1.10.1`
-
-Concrete example, I want to bump the hyperkube version from 1.9.3 to 1.10.1:
-```json
-{
-  "systemd": {
-    "units": [
-      {
-        "enabled": true,
-        "name": "pupernetes.service",
-        "contents": "[Unit]\nDescription=Run pupernetes\nRequires=setup-pupernetes.service docker.service\nAfter=setup-pupernetes.service docker.service\n\n[Service]\nEnvironment=SUDO_USER=core\nExecStart=/opt/bin/pupernetes daemon run /opt/sandbox --hyperkube-version 1.9.3 --kubectl-link /opt/bin/kubectl -v 5 --timeout 48h\nRestart=on-failure\nRestartSec=5\n\n[Install]\nWantedBy=multi-user.target\n"
-      }
-    ]
-  }
-}
-```
-
-Become:
-```json
-{
-  "systemd": {
-    "units": [
-      {
-        "enabled": true,
-        "name": "pupernetes.service",
-        "contents": "[Unit]\nDescription=Run pupernetes\nRequires=setup-pupernetes.service docker.service\nAfter=setup-pupernetes.service docker.service\n\n[Service]\nEnvironment=SUDO_USER=core\nExecStart=/opt/bin/pupernetes daemon run /opt/sandbox --hyperkube-version 1.10.1 --kubectl-link /opt/bin/kubectl -v 5 --timeout 48h\nRestart=on-failure\nRestartSec=5\n\n[Install]\nWantedBy=multi-user.target\n"
-      }
-    ]
-  }
-}
-```
+Upgrade Kind version by changing version in `test/e2e/scripts/run-instance/10-setup-kind.sh`.
+By default Kind will use the latest stable Kubernetes known at the time of Kind release.
```

Comment:
any reason why we don't install the latest stable kind version?

Reply:
There is no straightforward way to get latest but I guess we could implement it. Also it'd change K8S version automatically.

Reply:
> Also it'd change K8S version automatically.

A good way to catch agent/k8s incompatibilities as soon as we can.

Not a blocker though, we can work o installing latest kind in a separate PR

Reply:
I also wondered how we could automatically get the latest `kind` version.
I found nothing smarter than this:
```
curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kubernetes-sigs/kind/releases | jq -r '.[0].name'
v0.11.1
```
But maybe it’s good enough.

Reply:
I think it could be misleading when there's a patch release after a major release but it should be mostly okay

Reply:
Agreed on the concern!
I checked the order of the GitHub releases and it seems to be ordered. I.e.: I couldn’t find any patch release after a bigger major release:

```
curl -s -H "Accept: application/vnd.github.v3+json" https://api.github.com/repos/kubernetes-sigs/kind/releases | jq -r '.[] | .name'
v0.11.1
v0.11.0
v0.10.0
v0.9.0
v0.8.1
v0.8.0
0.7.0
v0.6.1
v0.6.0
v0.5.1
v0.5.0


0.2.1

0.1.0 - The second alpha release
0.0.1 - The first alpha test-release
```

But that’s very dependent on the project *habits*. And habits change.